### DISCUSSION TITLE
There is no README?

### DISCUSSION THREAD

**Discussion by marcellopato** [lshbluesky/Samsung-NT750XDA-KF59U-Hackintosh#27] [General]
Can't find a **HOW TO** or similar anywhere. There is a step-by-step tutorial on how to use these files?

**Comment by jukfiuune** [comment]
People are very hard at work, trying to fix the iris xe graphics, so no one will help you (from the devs). If you want i can help with your laptop. I and some other fellas have created a group just for this. Its in google chats, so ill need your gmail to invite you.

**Reply by jukfiuune** [reply]
I saw your email lol, ill send an invite. To accept it look in your inbox

**Reply by shady-milad** [reply]
Can u help me
